source/codegen/metadata/nirfmxgsm/config.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+config = {
+    'api_version': '21.0.0',
```

Comment:
[Minor] Is this the right version of nirfmxgsm you're pulling in here? This gets pulled in as a comment at the top of the generated .proto file.

Reply:
I changed the version to 23.0.0 in commit 888d28ec40e24341c1968e9deddf4901c236ab5c